### PR TITLE
Translate installation.mdx to Spanish

### PR DESCRIPTION
**Pull request by lilianabs**
# What does this PR do?

Adds the translation of [installation.mdx](https://github.com/huggingface/transformers/blob/master/docs/source/installation.mdx) to Spanish.

Fixes # [15947](https://github.com/huggingface/transformers/issues/15947) (issue)

## Before submitting
- [✓] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [✓] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

### CODE REVIEW

**Comment by omarespejel** [comment]
Thanks @lilianabs! 🤗

File: docs/source_es/installation.mdx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+</Tip>
+
+
+## Modo Offline
+
+🤗 Transformers puede ejecutarse en un ambiente detrás de un firewall o fuera de línea (Offline) utilizando archivos locales. Para habilitar éste comportamiento, declara la variable de ambiente `HF_DATASETS_OFFLINE=1`.
+
+<Tip>
+
+Puedes añadir  [🤗 Datasets](https://huggingface.co/docs/datasets/) al flujo de entrenamiento offline declarando la variable de ambiente  `HF_DATASETS_OFFLINE=1`.
```
Comment (omarespejel) [inline docs/source_es/installation.mdx:157]:
```suggestion
Puedes añadir [🤗 Datasets](https://huggingface.co/docs/datasets/) al flujo de entrenamiento offline declarando la variable de ambiente  `HF_DATASETS_OFFLINE=1`.
```

**Review by omarespejel — CHANGES_REQUESTED** [review]

**Comment by lilianabs** [comment]
@omarespejel all of the changes have been added, thank you! :)

**Review by omarespejel — APPROVED** [review]

**Comment by omarespejel** [comment]
Thank you @lilianabs! 🤗.  @sgugger LGTM 👍

**Review by sgugger — APPROVED** [review]
Thanks a lot for your PR!

**Pull request merged**